### ISSUE TITLE
Speed up search times

### ISSUE THREAD

**Issue by mjbriggs** [reynoldsnlp/flair#8]

**Comment by reynoldsnlp** [comment]
The biggest thing for this is saving a trained Random Forest model. We should make it delete the model on initialization, then build it in the `catch` block after `try`ing to access the saved model.

**Comment by reynoldsnlp** [comment]
#12 might also help with this. I'm sure disk IO slows things down at least a little.

**Comment by reynoldsnlp** [comment]
> The biggest thing for this is saving a trained Random Forest model. We should make it delete the model on initialization, then build it in the `catch` block after `try`ing to access the saved model.

[This stackoverflow answer](https://stackoverflow.com/questions/33556543/how-to-save-model-and-apply-it-on-a-test-dataset-on-java) will help. (quoted below)

> For versions greater than 3.5.5 it's as simple as:
> 
> J48 j48 = new J48();
> j48.buildClassifier(train);
> weka.core.SerializationHelper.write("/some/where/j48.model", j48);
> And loading is just:
> 
> Classifier cls = (Classifier) weka.core.SerializationHelper.read("/some/where/j48.model");
> See here for more info and how to do it in legacy versions!

**Comment by mjbriggs** [comment]
I have it set up so the server checks to see if the RandomForest model exists, if it does not then the server builds it (this operation takes a few seconds). I also have changed the raft package to use filestreams and inputstreams instead of creating files for each query. This did result in a speed up of arabic search times. I did not close this issue because I did not know if this speed up was sufficient. Additionally I did not know if we wanted to work on speeding up the other language searches.
